### PR TITLE
doc: CustomChainConfig blockExplorer -> blockExplorerUrl

### PR DESCRIPTION
**Pull request by Taewa**
If I understand correctly, `CustomChainConfig` 's `blockExplorer` no longer exists but `blockExplorerUrl` as an optional.
So I updated all related document.

CustomChainConfig : https://github.com/Web3Auth/web3auth-web/blob/master/packages/base/src/chain/IChainInterface.ts

### CODE REVIEW

**Comment by Taewa** [comment]
Hi @shahbaz17 
Do I have to assign anyone in order to kick start the PR? Or just wait to be reviewed?

**Comment by Taewa** [comment]
Hi @rtomas 
I wonder if I need some additional steps to request PR?

File: docs/sdk/core-kit/sfa-react-native/usage.mdx [hunk 1]
```diff
@@ -249,7 +249,7 @@ declare type CustomChainConfig = {
   /**
    * Url of the block explorer
    */
-  blockExplorer: string;
+  blockExplorerUrl?: string;
```
Comment (ihsraham) [inline docs/sdk/core-kit/sfa-react-native/usage.mdx:252]: Hey @Taewa,
Great job with the PR, and thanks for your contribution. But the SFA React native SDK hasn't been updated to use the updated ChainConfig. Once it's done, feel free to raise a PR again. For now, switch this back to blockExplorer. Make these changes and I'll be glad to merge your PR and mark your first contribution here. Thanks!

**Review by ihsraham — CHANGES_REQUESTED** [review]

**Comment by avious00** [comment]
hey @ihsraham the footer docs links are broken too: 
under Product
[Plug & Play](https://web3auth.io/docs/pnp/introduction)
[Core Kit](https://web3auth.io/docs/core-kit/introduction) 

it's going to the old /sdk/pnp/ path but the new url is https://web3auth.io/docs/examples?product=Plug+and+Play

**Pull request closed without merging**